### ISSUE TITLE
SPL Form Letter - Korean

### ISSUE THREAD

**Issue by blaughman** [usdoj-crt/crt-portal-management#1017]

**Comment by blaughman** [comment]
[1368286.SPL.Korean.final.docx](https://github.com/usdoj-crt/crt-portal-management/files/7009315/1368286.SPL.Korean.final.docx)

FBI approved version

**Comment by blaughman** [comment]
We will need the highlighted text removed from this form letter.  The phrases highlighted are preceded by their translations in the document.  This will not add any value to a recipient and only cause confusion.  We can chalk this up to a misunderstanding with the linguist that prepared the document.
[Korean Edits Needed.docx](https://github.com/usdoj-crt/crt-portal-management/files/7099778/Korean.Edits.Needed.docx)
@ilodidoj @billyhunt @louh @randyabramson @emilycryan

**Comment by billyhunt** [comment]
OK, I will create a fix!

**Comment by billyhunt** [comment]
@blaughman do you know if those words to be removed should be replaced with spaces or not?

**Comment by billyhunt** [comment]
I wonder if we should get them to create a new version as it should be. thoughts @blaughman?

**Comment by blaughman** [comment]
I just compared this to the English version.  There are no instances where Special Litigation Section or Civil Rights Division are followed by any punctuation.  @randyabramson @billyhunt Thoughts?
[SPL.Form.Letter.7.with.Header.Edit (4).docx](https://github.com/usdoj-crt/crt-portal-management/files/7100194/SPL.Form.Letter.7.with.Header.Edit.4.docx)

**Comment by billyhunt** [comment]
I was thinking more about spacing.  Should there be a space between where "(Special Litigation Section)" was or not?

**Comment by billyhunt** [comment]
Looks like we don't need spaces.  I will get working on the PR.

**Comment by blaughman** [comment]
@billyhunt I spoke with Randy and Imran regarding this letter.  We can leave the English wording in this particular form letter.  I would rather get this deployed to production and we can check with the linguist at a later time if we still feel this is an issue.

**Comment by billyhunt** [comment]
Sounds good!

**Comment by billyhunt** [comment]
Going to release as is wait on official language translation to create new card and update.  CC @blaughman @ilodidoj @louh
